data/organisations.json

```diff
@@ -3,12 +3,14 @@
     "$schema": "https://ils.rero.ch/schema/organisations/organisation-v0.0.1.json",
     "address": "Via Challand 132, 11100 Aosta",
     "name": "R\u00e9seau des biblioth\u00e8ques du Canton d'Aoste",
-    "code": "aoste"
+    "code": "aoste",
+    "default_currency": "CHF"
```

Comment:
**Aosta** is in Italy.  Should be **EUR**!

Reply:
You're right 👍therefore it sn't very critical